ci/scripts/wait-for-deploy.js

```diff
@@ -41,10 +42,19 @@ async function run () {
       const { taskDefinition, rolloutState, rolloutStateReason } = uncompletedDeployments[0]
       const deploymentDetails = {
         taskDefinition,
-        deploymentStatus: rolloutState,
-        appVersion,
-        nginxProxyVersion,
-        telegrafVersion
+        deploymentStatus: rolloutState
+      }
+      if (appVersion) {
+        deploymentDetails.appVersion = appVersion
+      }
+      if (nginxProxyVersion) {
+        deploymentDetails.nginxProxyVersion = nginxProxyVersion
+      }
+      if (telegrafVersion) {
+        deploymentDetails.telegrafVersion = telegrafVersion
+      }
+      if (carbonRelayVersion) {
+        deploymentDetails.carbonRelayVersion = carbonRelayVersion
```

Comment:
Nice change for these - should mean we can reuse for the Notifications app as well 👍